examples/ic_microcontroller/STM32F302R6T6TR.json

```diff
@@ -8,13 +8,13 @@
       "STM32F302R6T6xxx"
     ],
     "sourceDatasheetID": {
-      "publishedDate": "March2020",
-      "version": "Rev 8",
+      "publishedDate": "2020-03-00",
```

Comment:
Let's go with 2020-03-01, so that it's a real date

Reply:
yeah makes sense